### PR TITLE
Rspec tests for model: User

### PR DESCRIPTION
**Pull request by Philippoes**
# Pivotal Tracker story: https://www.pivotaltracker.com/story/show/136240359

# Changes proposed in this pull request:

This PR will add rspec tests for the user model. It will test two mailboxer methods, it will also check that the DB has a name column, and lastly validate that the user name can't be empty.

# What I have learned working on this feature:

How to test a model with methods

### Ready for review!

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -6,4 +6,22 @@
       expect(FactoryGirl.create(:user)).to be_valid
     end
   end
+
+  describe 'user' do
+    it 'the mailbox name should be subject.name' do
```
Comment (AmberWilkie) [inline spec/models/user_spec.rb]: Can you write this in a way that makes more sense to a human?

Comment (tochman) [inline spec/models/user_spec.rb]: please refer to methods using `#` 

```
it '#mailbox_name should return subject.name' do
```
for instance...

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -6,4 +6,22 @@
...
+      expect(subject.mailboxer_name).to eq subject.name
+    end
+
+    it 'the mailbox email should be subject.email' do
```
Comment (tochman) [inline spec/models/user_spec.rb]: Same as above

Comment (AmberWilkie) [inline spec/models/user_spec.rb]: Same thing.

**Review by AmberWilkie — COMMENTED** [review]
Little changes, otherwise everything looks fine.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -6,4 +6,22 @@
       expect(FactoryGirl.create(:user)).to be_valid
     end
   end
+
+  describe 'user' do
```
Comment (tochman) [inline spec/models/user_spec.rb]: `user` or `Mailboxer methods`?

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -6,4 +6,22 @@
...
+      expect(subject.mailboxer_email(subject.email)).to eq subject.email
+    end
+  end
+
+  describe 'DB table' do
+    it { is_expected.to have_db_column :name }
+  end
+
+  describe 'validations' do
+    it { is_expected.to validate_presence_of :name }
```
Comment (tochman) [inline spec/models/user_spec.rb]: If I'm not mistaken the name is also unique, right? Can't see a test for that.

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -6,4 +6,22 @@
...
+      expect(subject.mailboxer_name).to eq subject.name
+    end
+
+    it 'the mailbox email should be subject.email' do
+      expect(subject.mailboxer_email(subject.email)).to eq subject.email
+    end
+  end
+
+  describe 'DB table' do
+    it { is_expected.to have_db_column :name }
```
Comment (tochman) [inline spec/models/user_spec.rb]: I would also check for the presence of `email` collumn

**Review by tochman — CHANGES_REQUESTED** [review]

**Review by tochman — APPROVED** [review]

**Pull request merged**